Source/LuaBridge/detail/Expected.h

```diff
@@ -1265,7 +1266,7 @@ class Expected : public detail::ExpectedBase<T, E, std::is_copy_constructible_v<
     {
 #if LUABRIDGE_HAS_EXCEPTIONS
         if (!hasValue())
-            throw BadExpectedAccess<E>(error());
+            throw BadExpectedAccess<E>(error(), error().message());
```

Comment:
E is a templated class and can be anything. Here you are assuming is has a "message" method. We should solve it differently, using a detector on the `E::message` method (or eventually a `to_string(E)`) free function on the type if defined, inside the BadExpectedAccess constructor.